instrumentation/cassandra-datastax-4.0.0/src/test/java/com/nr/agent/instrumentation/cassandra/CassandraInstrumented.java

```diff
@@ -39,7 +40,9 @@
 import static org.junit.Assert.assertNotNull;
 
 // Issue when running cassandra unit on Java 9+ - https://github.com/jsevellec/cassandra-unit/issues/249
-@Category({Java9IncompatibleTest.class, Java10IncompatibleTest.class, Java11IncompatibleTest.class, Java12IncompatibleTest.class, Java13IncompatibleTest.class, Java14IncompatibleTest.class,
+// Many OpenJ9 and Cassandra issues e.g. :https://github.com/jsevellec/cassandra-unit/issues/295
```

Comment:
From what I can tell, anyone attempting to use cassandra with OpenJ9 is already having issues.